pkgs/by-name/ve/verovio/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  cmake,
+  git,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "verovio";
+  version = "4.2.0";
+
+  src = fetchFromGitHub {
+    owner = "rism-digital";
+    repo = "verovio";
+    rev = "version-${finalAttrs.version}";
+    hash = "sha256-PjBtnt46pHV3rDlwltVxfUMlgNoZGnI9CfnFtk2/j3s=";
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    git
+    makeWrapper
+  ];
+
+  dontConfigure = true;
+
+  buildPhase = ''
+    runHook preBuild
+
+    cd tools
+    patchShebangs .
+    cmake ../cmake
+    make -j 8
+
+    runHook postBuild
+  '';
```

Comment:
I'm not sure about this part. I'm pretty sure that this can be simplified.

Reply:
This is directly from the [instructions for building from source in the Verovio docs](https://book.verovio.org/installing-or-building-from-sources/command-line.html), with an extra `patchShebangs` thrown in to patch the tools.

I just confirmed that the build still fails without that `patchShebangs`.